### ISSUE TITLE
Periodic Table not focusable with JAWS keys.

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/build-an-atom#469]
**Device**
Samsung Laptop
**OS**
Win 11
**Browser**
Chrome + Jaws
**Problem Description**
For https://github.com/phetsims/qa/issues/1339
On the first level of the game screen, when pressing tab to reach the periodic table focus sometimes goes to the start over button instead. Unsure of what the exact cause is, but the below steps seem to reproduce it fairly consistantly.
**Steps to Reproduce**
1. With Jaws, start the first game level
2. Use arrow keys to read through the text until you get to the periodic table
3. Press tab to start interaction

**Visuals**

https://github.com/user-attachments/assets/04f04c31-2812-45b2-b0c1-546058a7e4b7

**Comment by jbphet** [comment]
Weird.  @KatieWoe, just to be clear, are you saying that you are _only_ seeing this when running Jaws?

I've tried duplicating it using NVDA (which is the only screen reader I have installed) and through just playing around with the sim using the alt-input keys and new screen reader, and I'm not seeing anything at all like this.  We may need to set up a meeting between me and someone with Jaws installed to do some debugging.

**Comment by Nancy-Salpepi** [comment]
Met with @jbphet. This is also an issue with NVDA. He is going to spend some time looking into it, but it might have to wait for @jessegreenberg.

**Comment by jbphet** [comment]
I have just tested game challenge navigation using JAWS version 2026.2512.50 on my Windows 11 machine (Dell Precision 7560), and many of the other challenges don't work either with respect to navigating using the arrow keys.

**Comment by terracoda** [comment]
The issue goes a little deeper than non-automatic activation of forms mode.

The Periodic Table is not present in the PDOM at all for reading. There is only the words "Periodic Table." There is also no description of the periodic table, 7 by 18 cell grid, and no indication of where the element is in the Periodic Table once an element is selected.

This interaction is currently implemented as a two-step process, but it is not clear it is a two-step process:

- Step 1: move to a custom navigable group (with role application) with no preselection of a group member and no representation of any group members;
- Step 2:  once navigating with arrow keys, auto-selection of each group member 1 at a time and no indication of where your selected element is in the Table.

Question/Challenge:
How to make the interaction accessible without a readable table in the PDOM.

**Comment by terracoda** [comment]
Note: We may not need a visual hint because Space, Enter and Arrows are all working when the RT gets focus.

**Comment by terracoda** [comment]
<img width="437" height="368" alt="Image" src="https://github.com/user-attachments/assets/c1590fd1-9daa-41b1-8161-cc8c9b674c5f" />

### UPDATES to Descriptions
- [ ] **Change:** "Meaningfully organizes the chemical symbols of its 91 elements. The upper 3 rows have several empty cells creating a u-shaped appearance for the table." **To:** "91 chemical symbols meaningfully organized in a 7 by 18 navigable table."
- [ ] Remove old help text: "Use WASD keys to select element on Periodic Table. Then look for the radio buttons, Neutral Atom or Ion, to complete your answer." 
- [ ] Remove the question: "Is the selected element a neutral atom or ion?" We can communicate this as the heading of the radiogroup.
- [ ] Update radiogroup heading to: "Is it a Neutral Atom or Ion?"
- [ ] Remove or update help text for the radiogroup. **Current help text:** "Select whether the specified atom is a neutral atom or ion. Then check your answer." If hep text is needed change **change to:** "Determine neutral atom or ion."

**Comment by jbphet** [comment]
@Nancy-Salpepi and @arouinfar - I've implemented the things we discussed in this morning's meeting.  You can see a summary of what has been changed by reviewing the list of commits just above this comment.  Please review the behavior and let me know if this matches your understanding of what we discussed and if you feel like there are any problems that should be addressed before moving on.

**Comment by Nancy-Salpepi** [comment]
@jbphet the periodic table is appearing in the description for the level 1 game button:

<img width="1084" height="219" alt="Image" src="https://github.com/user-attachments/assets/be3df379-65f6-4fb3-8e27-800298d47d2c" />

<img width="512" height="119" alt="Image" src="https://github.com/user-attachments/assets/9e2ebbc7-ee8b-4289-9c0c-7d287ab8502c" />

**Comment by jbphet** [comment]
I made the fixes above.  I think I'll leave this open and have it verified in the upcoming RC.
